### PR TITLE
Merge to dev - protoc compiler information

### PR DESCRIPTION
**Pull request by sunithasomanna**
Merge to dev - Added protoc compiler information in GS Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3768,4 +3769,92 @@ These checks run in parallel and take approximately 9 minutes to complete.
...
+
+   2. Add the following content to it:
+   
+      ```
+      #!/bin/bash -x
+      ## Generting protoc
+      
+      protos=("account" "aggregator" "auth" "chassis" "events" "fabrics" "managers" "role" "session" "systems" "task" "telemetry" "update" "compositionservice" "licenses")
+      for str in ${protos[@]}; do
+      proto_path="$repoFolder/lib-utilities/proto/$str"
```
Comment (ashraf-vazeer) [inline README.md:3844]: Value for  $repoFolder has to be added

Comment (sunithasomanna) [reply README.md:3844]: done

**Comment by sunithasomanna** [comment]
Closing this and raising a new PR to accommodate Ashraf's feedback from sub-branch. 
https://github.com/ODIM-Project/ODIM/pull/917

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md:3844]: done on #917

**Pull request closed without merging**